src/bluecadet/touch/drivers/MultiNativeTouchDriver.cpp

```diff
@@ -0,0 +1,160 @@
+#include "MultiNativeTouchDriver.h"
+
+using namespace ci::app;
+
+namespace bluecadet {
+namespace touch {
+namespace drivers {
+
+ci::signals::Signal<void(ci::vec2, int)>  signalOnTouchAdded;
+ci::signals::Signal<void(ci::vec2, int)>& getSignalOnTouchAdded() { return signalOnTouchAdded; }
+ci::signals::Signal<void(ci::vec2, int)>  signalOnTouchUpdated;
+ci::signals::Signal<void(ci::vec2, int)>& getSignalOnTouchUpdated() { return signalOnTouchUpdated; }
+ci::signals::Signal<void(ci::vec2, int)>  signalOnTouchRemoved;
+ci::signals::Signal<void(ci::vec2, int)>& getSignalOnTouchRemoved() { return signalOnTouchRemoved; }
+
+GET_POINTER_INFO			GetPointerInfo;
+GET_POINTER_TOUCH_INFO		GetPointerTouchInfo;
+GET_POINTER_PEN_INFO		GetPointerPenInfo;
+
+HWND						_currentWindow;
+LONG_PTR					_oldWindowProc;
+
+extern "C"
```

Comment:
@shi-weili is there a way to make this part of the `MultiNativeTouchDriver` class as static methods instead of just the namespace? Otherwise those signals are just kind of floating around publicly (even without the accessors).